src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/package.json

```diff
@@ -339,7 +339,13 @@
             {
                 "language": "aspnetcorerazor",
                 "scopeName": "text.aspnetcorerazor",
-                "path": "./syntaxes/aspnetcorerazor.tmLanguage.json"
+                "path": "./syntaxes/aspnetcorerazor.tmLanguage.json",
+                "embeddedLanguages": {
+                  "source.cs": "csharp",
```

Comment:
Out of curiosity, what is this mapping for, i.e. why do we need to map "source.cs" to "csharp" here instead of just listing "source.cs"? It seems like in the tmLanguage.json files only "source.cs" is specified, does specifying "csharp" also do something? (Hopefully my question is making sense lol.)

Reply:
Great question! So in VSCode land they have [built-in language-configuration.json's](https://github.com/microsoft/vscode/tree/main/extensions) for nearly every language so you don't technically need extensions to edit those langs. When I specify "csharp" here what I'm referring to is the general "csharp" language [defined by VSCode](https://github.com/microsoft/vscode/blob/813deed41ccef3873b3e30eb4e646fcba6991ed7/extensions/csharp/package.json#L17). Same with the other bits. Ultimately this means we don't need to re-ship all of the sub-language language-configuraiton.json's instead we can use the ones already included in VSCode 😄